README.md

```diff
@@ -227,3 +228,29 @@ These may be defined as a hash passed to ::squid
 * `scheme` the scheme used for authentication must be defined
 * `entries` An array of entries, multiple members results in multiple lines in squid.conf
 * `order` by default is '40'
+
+### Defined Type squid::extra\_config\_section
+Squid has a large number of configuration directives.  Not all of these have been exposed individually in this module.  For those that haven't, the `extra_config_section` defined type can be used.
+
+```puppet
+squid::extra_config_section {'mail settings':
+  order          => '60',
+  config_entries => {
```

Comment:
Really?  Like this?
```puppet
squid::extra_config_section {'mail settings':
  order            => '60',
  config_entries   => {
    'mail_from'    => 'squid@example.com',
    'mail_program' => 'mail',
  },
}
```
That would fail puppet-lint if you wrote it like that in a manifest.